Create HetznerUbuntuSetup.md

heathriel opened the pull request:
Step-by-step guide for someone who is starting from scratch on this project with a bare server.

﻿Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.



## Type of change

Please delete options that are not relevant.
- [x] Documentation update  
  

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
##


### **Test Configuration**:
##


## Checklist:

- [ x] My code follows the style guidelines of this project

heathriel committed:
Create HetznerUbuntuSetup.md

Step-by-step guide for someone who is starting from scratch on this project with a bare server.

ClaraLeigh reviewed (changes requested):
Looks great! Any chance you could also change the main README.md to have a link to this? There is already a "Cloud Deployment" heading there

heathriel committed:
Updated Readme & Heroku

I submitted the original Heroku.md (to the discord) and they are way out of date. Just corrected them, moved the Hetzner file to the cloud deploy, and updated the readme to point to the file.

heathriel commented:
Made changes. closing and reoopening with new updates.

heathriel committed:
Update HetznerUbuntuSetup.md

heathriel committed:
Update README.md

fuegovic reviewed (commented):
I see you updated the heroku instruction at the same time, thanks!
